controllers/secrets/externalsecret_controller_test.go

```diff
@@ -130,6 +130,25 @@ var _ = Describe("ExternalsecretController", func() {
 
 		})
 
+		It("Should Fail creating new secret for CR when the backend fails", func() {
```

Comment:
Please change to "Should fail when backend with empty key is passed"

Reply:
It should be updated now.